src/pages/difficulty.tsx

```diff
@@ -85,6 +33,24 @@ const DifficultyPage = (): React.ReactElement => {
                     </li>
                 ))}
             </ul>
+
+            <h2>Tool Kit DCs</h2>
+            <div>
+                {DC_TOOLS.map(tool => (
+                    <div key={tool.name}>
+                        <h2>{tool.name}</h2>
```

Comment:
I think his should be an `h3` instead of an `h2` since the block "heading" is an `h2`